orderly_web/config.py

```diff
@@ -139,6 +139,13 @@ def __init__(self, path, dat):
         self.sass_variables = config_string(dat,
                                             ["web", "sass_variables"],
                                             True)
+        self.logo_path = config_string(dat, ["web", "logo"], True)
+        if self.logo_path is not None:
+            self.logo_path = os.path.abspath(
```

Comment:
you use this pattern in two places now - could you factor into a little helper (absolute path relative to the orderly-web root) and then later we can centralise the validation that this is in fact a relative path